be/src/runtime/tablet_writer_mgr.h

```diff
@@ -92,6 +96,15 @@ class TabletWriterMgr {
         TabletsChannelKeyHasher> _tablets_channels;
 
     Cache* _lastest_success_channel = nullptr;
+
+    std::unordered_map<TabletsChannelKey, time_t,TabletsChannelKeyHasher> _key_time_map;
```

Comment:
I think it's better to add a `last_updated_time` member in `TabletsChannel `, then there is no need to maintain two map.

And when there is a new request referring to this tablets, then update this field.

Then we can compare current wall time to this `last_updated_time` to see if it has been exceeded the config timeout.

And I think this timeout can be some shorter time, such as 10 minutes

Reply:
OK